app/models.py

```diff
@@ -0,0 +1,56 @@
+from sqlalchemy import Column, String, Boolean, DateTime
+from sqlalchemy.orm import Mapped, mapped_column
+from sqlalchemy.dialects.postgresql import UUID
+import uuid
+from datetime import datetime
+from app.db.database import Base
+from pydantic import BaseModel, EmailStr, Field
+from datetime import datetime
+
+
+class User(Base):
+    __tablename__ = "users"
+
+    id: Mapped[uuid.UUID] = mapped_column(
+        UUID(as_uuid=True), primary_key=True, default=uuid.uuid4
+    )
+    username: Mapped[str] = mapped_column(String, nullable=False)
+    email: Mapped[str] = mapped_column(String, unique=True, nullable=False)
+    hashed_password: Mapped[str] = mapped_column(String, nullable=False)
+    is_admin: Mapped[bool] = mapped_column(Boolean, default=False)
+    is_active: Mapped[bool] = mapped_column(Boolean, default=True)
+    created_at: Mapped[datetime] = mapped_column(DateTime, default=datetime.utcnow)
+    updated_at: Mapped[datetime] = mapped_column(
+        DateTime, default=datetime.utcnow, onupdate=datetime.utcnow
+    )
+
+
+class UserDTO(BaseModel):
```

Comment:
What does DTO mean? Please make the name clearer.

Reply:
Sry, It mean Data transfer model ( same mean for DataResponse or DataCreate or DataUpdate ) but it's the global meaning .